.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💬 Help & Questions
+    url: https://github.com/sequelize/sequelize/discussions
+    about: The issue tracker is for bug reports and feature requests. Use GitHub Discussions for question and help requests.
```

Comment:
```suggestion
    about: The issue tracker is for bug reports and feature requests on our documentation website. Use GitHub Discussions for question and help requests on Sequelize itself.
```

Reply:
The part about bugs and feature requests doesn't make sense for a documentation repo. It's for documentation issues